### ISSUE TITLE
Update PyVimeo with git master

### ISSUE THREAD

**Issue by MatthiasWiesner** [vimeo/vimeo.py#92]
The current version of PyVimeo (installed with pip) throws an import error:
```
>>> import vimeo
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/mwiesner/.virtualenvs/vimeoupload/lib/python2.7/site-packages/vimeo/__init__.py", line 8, in <module>
    from .client import VimeoClient
  File "/Users/mwiesner/.virtualenvs/vimeoupload/lib/python2.7/site-packages/vimeo/client.py", line 12, in <module>
    from .exceptions import APIRateLimitExededFailure
ImportError: cannot import name APIRateLimitExededFailure
```
simply a typo `APIRateLimitExededFailure`

**Comment by pg1671** [comment]
This appears to be included in the latest PyPi release 0.3.7
having to update requirements.txt to force 0.3.6 until fixed

**Comment by agilefox** [comment]
This should be fixed on 0.3.8. Sorry!
